streamlog/streamlog

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Print the Field System log to stdout
+set -euo pipefail
+
+if [[ -z ${FS_DISPLAY_SERVER:-} ]]; then
+    if ! fsserver status > /dev/null; then echo fsserver not running; exit 1 ; fi
+    if ! fsserver fs status > /dev/null; then echo fs not running; exit 1; fi
+    FS_SERVER_URL_BASE=$(awk '/FS_SERVER_URL_BASE/ {gsub(/"/, "", $3); print $3}' /usr2/fs/include/params.h)
```

Comment:
For pre-beta1, do I replace this line with:

ssub -s -w ipc:///var/run/fsserver/windows/fs/{pub,rep}

or something else?

Reply:
No, delete this line for pre -beta1

Reply:
I hadn't planned on backporting this, partly as the display stream is a (slightly) different format. I figure users can just upgrade to get `streamlog` and the new server features at the same time.